### DISCUSSION TITLE
Error on deploy new project

### DISCUSSION THREAD

**Discussion by alvesadolfo** [vercel/community#6856] [Account Issue] [answered]
### Question

My plan is Hobby, but I have 2 projects and do not have permission to import the third project.

Message Error: "You are attempting to import a Git repository from a GitHub organization, GitLab group, or Bitbucket team. Start a Pro Trial or upgrade to Pro."

**Comment by wilrnh** [comment]
i also have this issue, but i only have 1 project

**Comment by versecafe** [comment]
@alvesadolfo @wilrnh Is the repository you're attempting to import part of your github account itself, or is it from an organization you're a member of?

**Reply by wilrnh** [reply]
It is a part of a github org. However, the 1 project that I have currently connected is also part of the same github org: https://github.com/flushingtech/website

**Reply by versecafe** [reply]
If you're using a hobby plan you can't link projects from a github organization even if you are the only member.

**Reply by wilrnh** [reply]
Thanks @versecafe but it seems like [flushingtech/website](https://github.com/flushingtech/website) is part of an org, yet its being deployed just fine https://vercel.com/wilrnh-projects/flushing-tech-website/AwFRXbN9992arZkKR4FjpYTg3V5E. I wonder if there is a limit of 1 org project on the hobby plan?

**Reply by versecafe** [reply]
Is the working one opensource?

**Reply by wilrnh** [reply]
Hey @versecafe, yes the working project is opensource, but so are all the projects I'd like to add.

Is being opensource an exception to the github organization rule for hobby plans?

**Reply by leog** [reply]
I have the same doubt @versecafe

**Reply by versecafe** [reply]
Being opensource is not a requirement for the Hobby plan but for a while only opensource projects would auto deploy on commits from users not in the Vercel org, I don't know if that's still true or if there is legacy behaviour for this that keeps it working in some projects.

**Comment by BigTava** [comment]
So is there any workaround for this?

**Comment by amyegan** [comment, marked as answer]
Hi there! The Hobby plan supports public and private git repositories hosted in a personal account scope. Pro teams have additional [collaboration features](https://vercel.com/docs/getting-started-with-vercel/collaborate), including the ability for team members to deploy from GitHub organizations, GitLab groups, and non-personal Bitbucket workspaces. 

You can solve the error by moving your repo to a personal account instead of an organization or by [upgrading to Pro](https://vercel.com/docs/accounts/plans/pro#upgrading-to-pro). If you’re going to be regularly working with collaborators, the Pro plan is probably a better fit for your use case.

You can find more info in this guide: [Why is Vercel not deploying my latest commits?](https://vercel.com/guides/why-aren-t-commits-triggering-deployments-on-vercel)

**Reply by VPNAM-02** [reply]
how to move my repo to a personal account ? When vercel required create a team ?

**Reply by AlekseiAQ** [reply]
> how to move my repo to a personal account ? When vercel required create a team ?

https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-organization

[a comment by an automated account is not shown]
